src/Docker.DotNet/Microsoft.Net.Http.Client/BufferedReadStream.cs

```diff
@@ -159,7 +159,7 @@ public bool Peek(byte[] buffer, uint toPeek, out uint peeked, out uint available
 
     public async Task<string> ReadLineAsync(CancellationToken cancellationToken)
     {
-        var line = new StringBuilder(_buffer.Length);
+        var line = new StringBuilder();
```

Comment:
```suggestion
        var line = new StringBuilder(30);
```

Oh, good catch. You're right, I missed that after refactoring. I'd suggest choosing a slightly larger capacity than the default. This will prevent unnecessary increases. I quickly checked the headers we read for a simple configuration, and they're usually larger than the default capacity. WDYT?

Reply:
Yes, sounds good, changed it. 👍